apps/docs/src/.vitepress/config.ts

```diff
@@ -1,12 +1,8 @@
-import path from "node:path";
 import { fileURLToPath } from "node:url";
-import { defineConfig, loadEnv, type DefaultTheme } from "vitepress";
+import { defineConfig } from "vitepress";
 import packageJson from "../../../../packages/sit-onyx/package.json";
-import { getStorybookHost } from "./env";
-import { getStorybookSidebarFolders } from "./utils";
 
-const env = loadEnv("", path.join(process.cwd(), "src"));
-const STORYBOOK_HOST = getStorybookHost(env);
+const storybookLink = "https://storybook.onyx.schwarz" as const;
```

Comment:
if you want you could export that URL in a central file so if it changes we only have to adjust it once.

as soon as we got staging with the v1 release, this might be relevant?

Reply:
I guess we also would have to change it in all markdown files then :)
For now I wanted to keep it simple so we don't need to use env variables and vue bindings in markdown files